doc/source/getting_started/install.rst

```diff
@@ -245,6 +245,9 @@ Package                                                          Minimum support
 Recommended dependencies
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
+pandas recommends the following optional dependencies for performance gains. Can be managed via the
+``performance`` extra when defining the pandas requirement.
```

Comment:
For each of these sentences specifying which label installs the packages, could we more explicitly say "These dependencies can be specifically installed with `pandas[foo]`"